CardanoSharp.Blazor/CardanoSharp.Blazor.Components/WalletConnector.razor

```diff
@@ -209,10 +220,76 @@ else
         //set up initial _wallets state from above
         _selfReference = DotNetObjectReference.Create(this);
         _wallets = await _walletConnectorJs.Init(SupportedExtensions, _selfReference);
+        
         Initialized = true;
+
+        await WalletOnInitialized();
+
         return;
     }
 
+    private async Task WalletOnInitialized()
```

Comment:
please add Async as a method name suffix for async methods
the method name is also a bit too abstract, there is already a generic OnInitialized. in this method we are specifically doing stuff related to the persisted wallet key/state, so lets name it something like 'InitStoredWalletKeyAsync' or something along those lines.